articles/ros2_threat_model.md

```diff
@@ -2496,8 +2499,29 @@ to True</li>
     </td>
     <td class="warning">Risk is mitigated if DDS-Security is configured
 appropriately.</td>
-    <td> </td>
-  </tr>
+    <td>As of writing, XRCE only has implementations for UDP, TCP, and serial
+    communication. Adding implementations for TLS and DTLS would give a way to
+    provide security at the transport level in addition to the network level.</td>
+  </tr>
+  <tr>
+      <td class="danger">✓</td>
+      <td class="danger">✓</td>
+      <td class="danger">✓</td>
+      <td class="success">✘</td>
+      <td class="success">✘</td>
+      <td class="danger">✓</td>
+      <td>TurtleBot LIDAR measurements are saved to a remote location controlled by
+  the attacker. </td>
+      <td> Limiting access to the UDP ports used by the XRCE agent and client to
+        a specific user running the agent and client processes
```

Comment:
How would you do that? cgroups?

Reply:
I wasn't really sure on the details here. I've changed this to mention that the loopback interface should be sufficient